logs_sdk/test/opentelemetry/sdk/logs/log_record_test.rb

```diff
@@ -96,5 +96,86 @@
         assert_equal(args[:instrumentation_scope], log_record_data.instrumentation_scope)
       end
     end
+
+    describe 'attribute limits' do
+      it 'uses the limits set by the logger provider via the logger' do
+        skip 'need console log record exporter to be merged first'
```

Comment:
Remove this `skip` once #1687 is merged

```suggestion
        skip 'need console log record exporter to be merged first'
```